app/src/main/java/arctouch/upcomingmovies/utils/Utils.java

```diff
@@ -0,0 +1,31 @@
+package arctouch.upcomingmovies.utils;
+
+/**
+ * Utility Class.
+ * Stores useful variables to be used in distinct parts of the program.
+ *
+ * @author Cassio Espindola
+ */
+public class Utils {
```

Comment:
I think the name Utils is too generic, would be a good idea having these constants in an specific class with a better name or even keep these values in the gradle build config could be a good option.

Reply:
Thank you for the tip.